FEXHeaderUtils/FEXHeaderUtils/Filesystem.h

```diff
@@ -354,6 +354,10 @@ inline fextl::string GetFilename(const fextl::string& Path) {
   return PathToString(std::filesystem::path(Path).filename());
 }
 
+inline fextl::string GetFilename(std::string_view Path) {
+  return PathToString(std::filesystem::path(Path).filename());
```

Comment:
@bylaws Could you briefly weigh in on this: Would it be safe to return a sub-`string_view` of the input path here (for consistency with Linux), or are there weird cases for which `filename()` returns a filename that's not a substring of the input on Windows?

```suggestion
  auto Filename = PathToString(std::filesystem::path(Path).filename());
  return Path.substr(Path.size() - Materialized.size());
```